code/game/objects/items/robot/items/inflatable_tool.dm

```diff
@@ -0,0 +1,43 @@
+/obj/item/inflatable/cyborg
+	name = "inflatable wall"
+	desc = "A folded membrane which rapidly expands into a large cubical shape on activation."
+	var/power_use = 400
+	var/structure_type = /obj/structure/inflatable
+	var/delay = 1 SECONDS
+
+/obj/item/inflatable/cyborg/door
+	name = "inflatable door"
+	desc = "A folded membrane which rapidly expands into a simple door on activation."
+	icon_state = "folded_door"
+	power_use = 600
+	structure_type = /obj/structure/inflatable/door
+
+/obj/item/inflatable/cyborg/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>As a synthetic, you can restore them at a <b>cyborg recharger</b></span>"
+
+/obj/item/inflatable/cyborg/attack_self(mob/user)
+	if(locate(/obj/structure/inflatable) in get_turf(user))
+		to_chat(user, "<span class='warning'>There's already an inflatable wall!</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='warning'>There's already an inflatable structure!</span>")
```
Or written in another way too since it could be a door there.